### ISSUE TITLE
Buttons should have some tactile feedback

### ISSUE THREAD

**Issue by lhorace** [marazmista/radeon-profile#72]
Hi,

Some buttons, for example, for fantcontrol and overclocking settings, have no tactile feedback once the buttons are depressed that the settings have been applied. May I suggest an enhancement that once the setting(s) have been applied, that the button become disabled (or faded out) till a change is made on the form?

**Comment by mainmachine** [comment]
Do you mean visual feedback? Because I'd agree with that - maybe just a better visual indication of the button state.

**Comment by lhorace** [comment]
Yes, exactly, because I find myself pressing the button multiple times because I am not sure if the settings are applied.
